### DISCUSSION TITLE
Vuetify failed to decode Image in Faces

### DISCUSSION THREAD

**Discussion by Lagicrus** [photoprism/photoprism#3916] [Q&A]
PLEASE PROCEED ONLY IF YOU ARE ABSOLUTELY SURE THAT THIS IS NOT A TECHNICAL SUPPORT INCIDENT AND/OR POSSIBLY A PROBLEM WITH SOME OTHER SOFTWARE YOU ARE USING. VISIT <https://www.photoprism.app/kb/getting-support> TO LEARN MORE ABOUT OUR SUPPORT OPTIONS. THANK YOU FOR YOUR CAREFUL CONSIDERATION!

---------------------------------------------------------------------------

#### 1. What is not working as documented?

Please be as specific as possible and explain what part of the software is not [working as documented](https://docs.photoprism.app/). "No photos found" would not be detailed enough. Also, never report [known issues](https://docs.photoprism.app/known-issues/) or [features not yet implemented](https://github.com/photoprism/photoprism/issues) as bugs. Thank you!

When I navigate to the `people/new` section, the site hitches and then I get 400 warnings all the same reporting that
```
[Vuetify] Failed to decode image, trying to render anyway

src: /api/v1/t/fa433bd845e1b88256e240f76d2349dc26806b02-1550491160bc/3jk0if6o/tile_320
Original error: The source image cannot be decoded.

found in

---> <VImg>
       <VCard>
         <PPageFaces>
           <VTabItem>
             <VTabsItems>
               <ThemeProvider>
                 <VTabs>
                   <PPagePeople>
                     <VContent>
                       <VApp>
                         <PhotoPrism>
                           <Root>
```

#### 2. How can we reproduce it?

Steps to reproduce the behaviour:

1. Go to 'people/new'
2. Check dev console when you have pending images there
3. See warnings

When reporting an import, indexing, or performance issue, please include the number and type of pictures in your library, as well as any configuration options you have changed, such as for thumbnail quality.

#### 3. What behavior do you expect?

It not to hitch but this is probably due to all the warnings, so I'd expect not to have 400 warnings about image rendering.

#### 4. What could be the cause of your problem?

I don't know but at a guess it could be some metadata issue, but seeing as it is the faces that Photoprism extracted itself, I am unsure.

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

Would these be the faces that it is trying to render?

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: 231021-9abea5b55

(b) Database Type & Version: MariaDb

(c) Operating System Types & Versions: Windows 11

(d) Browser Types & Versions: Chrome 119.0.6045.160

(e) Ad Blockers, Browser Plugins, and/or Firewall Software? Yes but this shouldn't affect image rendering

#### 7. On what kind of device is PhotoPrism installed?

This is especially important if you are reporting a performance, import, or indexing issue. You can skip this if you're reporting a problem you found in our public demo, or if it's a completely unrelated issue, such as incorrect page layout.

(a) Device / Processor Type

(b) Physical Memory & Swap Space in GB: 32GB in Physical

(c) Storage Type: HDD

(d) Anything else that might be helpful to know?

*Always provide device, memory, and storage details if you have a backend, performance, import, or indexing issue.*

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN? Not yet

**Comment by graciousgrey** [comment]
Are the thumbnails in the other sections displayed properly? 

Can you you give us some more information, such as how the volumes section of your docker-compose file looks like and where the folders are located.

Do you see any related errors or warnings in the  [docker logs](https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3) after enabling debug logs?

**Comment by Lagicrus** [comment]
As far as I can tell, all the images in that section are displaying correctly and if I navigate to the API URL that it is complaining about, then in the browser, the image comes through with no issues.

I have Photoprism running on TrueNas Scale via the TrueCharts docker image. I have it mounting a local path instead of a PVC.
I don't think there are any permission errors going on as it managed to successfully index all the thousands of images, and thus far, I have yet to run into an issue with seeing any thumbnails for images.
I also set it to "Read Only" mode if that matters, as the "originals" folder that is mounted is synced via Syncthing.

And no, after enabling Debug mode. I see no errors or warnings in the Docker logs when I send a request to list the "new faces". I just get a `GET` request for it that takes 373ms

**Comment by graciousgrey** [comment]
Ok so there is no functional problem but errors on the dev console?

Do you also observe this on our public demo?

   https://demo.photoprism.app/library/people/new

**Comment by Lagicrus** [comment]
Correct. Warnings in the dev console. 
Part of me suspects that the 400 warnings also cause a performance dip but that's a guess

And nope. No errors in the demo.
Also if it helps, had a quick click through of the other sections, currently no other section causes the same warnings to appear. Only `faces/new`

**Comment by graciousgrey** [comment]
We do not observe this on our private instances either. So it's more likely not a general problem, but something related to your specific setup. It would be good to know if others are also experiencing this problem.

**Comment by lastzero** [comment]
@Lagicrus Do you have any browser plugins installed, e.g. for manual face tagging?

Another cause could be that the server returns an error or a placeholder SVG icon when a thumbnail of the face could not be found/created, e.g. because the original image doesn't exist anymore or there are file system permissions issues.

I'll convert this into a discussion until it's clear how to reproduce the issue and if it's something that needs to be resolved in our code. If so, we'll be happy to provide a fix!

**Comment by Lagicrus** [comment]
Hey, sorry for the delay @lastzero . I think the answer is no here. I don't have any plugins installed specifically for tagging faces in PhotoPrism or even knew there were plugins for it?
I have browser extensions yes, just none specifically for PhotoPrism?

Looking at the src URLs in the warnings, they do seem to resolve to an image of a face and not a placeholder
